test/integration/topic_browsing_test.rb

```diff
@@ -15,6 +15,23 @@ def oil_and_gas_topic_item(params = {})
     base.merge(params)
   end
 
+  it "is possible to visit the topic index page" do
+    content_store_has_item("/topic", oil_and_gas_topic_item.merge({
+      :links => {
+        "children" => [
+          {
+            "title" => "Oil and Gas",
+            "base_path" => "/topic/oil-and-gas",
+          },
+        ],
+      }
+    }))
+
+    visit "/topic"
+
+    assert page.has_content?("Oil and gas")
```

Comment:
This feels wrong. It would be better if this test could use more representative data.